### PR TITLE
Feature/code review

### PR DESCRIPTION
**Pull request by odeimaiz**

### CODE REVIEW

File: services/web-client/source/class/qxapp/Application.js [hunk 1]
```diff
@@ -45,41 +43,34 @@ qx.Class.define("qxapp.Application", {
...
+          width: "100%"
+        });
         // }
-      });
+      }, this);
 
-      doc.set({
-        backgroundColor: "dark-blue"
-      });
+      doc.set({});
```
Comment (pcrespov) [inline services/web-client/source/class/qxapp/Application.js]: There is no need to set Nothing

File: services/web-client/source/class/qxapp/components/workbench/SettingsView.js [hunk 2]
```diff
@@ -47,35 +46,34 @@ qx.Class.define("qxapp.components.workbench.SettingsView", {
...
-        scope.fireEvent("SettingsEditionDone");
-      }, scope);
+        this.fireEvent("SettingsEditionDone");
+      }, this);
     },
 
-    _InitSettings: function() {
-      this._settingsBox = new qx.ui.container.Composite(new qx.ui.layout.VBox(10));
-      this.add(this._settingsBox);
+    __InitSettings: function() {
```
Comment (pcrespov) [inline services/web-client/source/class/qxapp/components/workbench/SettingsView.js]: naming consistency? should not be ``__initSettings``

File: services/web-client/source/class/qxapp/Application.js (hunk 1 above)
Comment (odeimaiz) [reply services/web-client/source/class/qxapp/Application.js]: Danke

File: services/web-client/source/class/qxapp/components/workbench/SettingsView.js (hunk 2 above)
Comment (odeimaiz) [reply services/web-client/source/class/qxapp/components/workbench/SettingsView.js]: Danke

File: services/web-client/source/class/qxapp/wrappers/WebSocket.js [hunk 3]
```diff
@@ -126,60 +126,59 @@ qx.Class.define("qxapp.wrappers.WebSocket", {
...
+        this.on("reconnecting", function(ev) {
+          this.fireDataEvent("reconnecting", ev);
+        }, this);
+        this.on("reconnect_failed", function() {
+          this.fireEvent("reconnect_failed");
+        }, this);
+        this.on("error", function(ev) {
+          this.fireDataEvent("error", ev);
+        }, this);
+      }, this);
```
Comment (oetiker) [inline services/web-client/source/class/qxapp/wrappers/WebSocket.js]: I would write this as:

```javascript
[
'x',
'y'
].forEach( event => {
  this.on(event,ev=> {
      this.fireDataEvent(event,ev);
  },this)
},this);
```

Comment (odeimaiz) [reply services/web-client/source/class/qxapp/wrappers/WebSocket.js]: Nice, done

**Pull request merged**